app/controllers/application_controller.rb

```diff
@@ -23,7 +23,7 @@ def hub?
   end
 
   def current_intake
-    Intake.find_by_id(session[:intake_id])
+    current_client&.intake || Intake.find_by_id(session[:intake_id])
```

Comment:
not yet understanding how this relates to current_step - in other words, why couldn't it have been changed separately from this work?

Reply:
We're dumping existing clients with existing intakes back into the intake flow for the first time. Controllers like DocumentsController and QuestionsController inherit directly from ApplicationController and need access to the current_clients intake, or else the "intake" that we redirect them to is a new anonymous intake. I guess this could be done separately as part of another task, but my guess is there wasn't a use case where this needed to be updated and now there is.